nsls2forge_utils/auto_tick.py

```diff
@@ -696,9 +697,12 @@ def clean(include=None, exclude=None, yes=False):
                 yes = True
     print('Removing files...')
     for path in to_be_removed:
-        files = glob.glob(path)
+        files = glob.glob(path, recursive=True)
         for file in files:
-            os.remove(file)
+            try:
+                os.remove(file)
+            except IsADirectoryError:
+                shutil.rmtree(file)
```

Comment:
I think we just want to pass here.
```suggestion
                pass  # we want to preserve the directory structure, but not the files.
```